instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/builder/internal/DefaultHttpClientInstrumenterBuilder.java

```diff
@@ -222,6 +222,10 @@ public OpenTelemetry getOpenTelemetry() {
     return openTelemetry;
   }
 
+  public String getInstrumentationName() {
+    return instrumentationName;
+  }
```

Comment:
to me this method and `getOpenTelemetry` feel out of place in this class.

Reply:
added a `instrumenterBuilder` method instead and only exposed propagators of `OpenTelemetry`